CMakeLists.txt

```diff
@@ -2,14 +2,8 @@ cmake_minimum_required(VERSION 3.20)
 
 project(ja2)
 
-set(ProfilePath "${CMAKE_SOURCE_DIR}/CMakePresets.json")
-set(UserProfilePath "${CMAKE_SOURCE_DIR}/CMakeUserPresets.json")
-if(NOT DEFINED Languages AND NOT DEFINED Applications AND NOT EXISTS "${ProfilePath}" AND NOT EXISTS "${UserProfilePath}")
-  set(UserProfileTemplatePath "${CMAKE_SOURCE_DIR}/CMakeUserPresets-TEMPLATE.json")
-  file(READ "${UserProfileTemplatePath}" UserProfileTemplatContent)
-  file(WRITE "${UserProfilePath}" "${UserProfileTemplatContent}")
-  message( FATAL_ERROR "No existing profile was found, copied a profile template as CMakeUserProfile.json . Writing an error to trick Visual Studio into reloading the new profile automatically." )
-endif()
+include(cmake/CopyUserPresetTemplate.cmake)
```

Comment:
now I read by my unrealiabe source, that include is definitly not supported by VS2019, can you check?

Reply:
maybe that info is from an older VS 2019 version and they fixed it in the meantime dunnow

Reply:
include is supported since cmake 2.6, released 15 years ago.